src/zh/onemanhua/build.gradle

```
@@ -6,7 +6,7 @@ ext {
     extName = 'ColaManhua (OhManhua)'
```

```suggestion
    extName = 'COLAMANGA'
```
Change the name field inside source class too.